contracts/PeeranhaCommunityToken.sol

```diff
@@ -0,0 +1,138 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "./interfaces/IPeeranhaCommunityToken.sol";
+import "./interfaces/IPeeranhaCommunityTokenFactory.sol";
+import "./libraries/CommonLib.sol";
+import "./base/NativeMetaTransaction.sol";
+
+import "@openzeppelin/contracts-upgradeable/token/ERC20/extensions/IERC20MetadataUpgradeable.sol";
+
+// transfer: 
+//    payable(userAddress).transfer(amount);
+//    IERC20Upgradeable(tokenAddress).transfer(userAddress, 2);
+//
+// balace:
+//    userAddress.balance
+//    IERC20Upgradeable(tokenAddress).balanceOf(userAddress)
+
+
+contract PeeranhaCommunityToken is IPeeranhaCommunityToken, NativeMetaTransaction {  
+
+  // sumAccruedTokens -> free tokens (added - pool)
+  // sumSpentTokens -> active tokens tokens (pools - give reward)
+  // frezeTokens - not taked pool
+  struct CommunityToken {
+    string name;
+    string symbol;
+    address tokenAddress;
+    uint256 maxRewardPerPeriod;
+    uint256 activeUsersInPeriod;
+    uint256 maxRewardPerUser;
+    uint256 frezeTokens;
+    uint256 createTime;
+    address peeranhaCommunityTokenFactoryAddress;
+  }
+
+  struct CommunityTokenContainer {
+    CommunityToken info;
+    mapping(uint16 => uint256) periodPool;
```

Comment:
is it pool of ready to claim rewards? can we name it accordingly?
`readyToClaimRewardPool`